ext/standard/var.c

```diff
@@ -641,6 +938,26 @@ PHP_FUNCTION(var_export)
 }
 /* }}} */
 
+/* {{{ Outputs or returns a short string representation of a variable */
```

Comment:
```suggestion
/* {{{ Returns a short string representation of a variable */
```

(only not sure about "short" since the rename)

Reply:
Done